### PR TITLE
Update catalog client to include positive grant features

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

File: docs/CatalogObjectGrantControllerApi.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
+# CatalogObjectGrantControllerApi
+
+All URIs are relative to *https://marouns-mbp:8080/catalog*
```
Comment (fviale) [inline docs/CatalogObjectGrantControllerApi.md]: why this url contains your name ?

Comment (marounkoussaifi) [reply docs/CatalogObjectGrantControllerApi.md]: ah weird. Let me check again.

Comment (marounkoussaifi) [reply docs/CatalogObjectGrantControllerApi.md]: It should be fixed now

**Review by aminelouati — APPROVED** [review]

**Review by fviale — APPROVED** [review]
LGTM

**Comment by fviale** [comment]
Retest this please

**Pull request merged**